schema.graphql

```diff
@@ -43493,6 +43493,52 @@ enum SquashMergeCommitTitle {
   PR_TITLE
 }
 
+"""
+Represents an SSH signature on a Commit or Tag.
+"""
+type SshSignature implements GitSignature {
```

Comment:
@timrogers it's odd that the type was added, but it's not used anywhere?

Reply:
Agreed - that doesn't seem weird. I have pinged the team at GitHub that introduced this. I'm going to merge this anyway, as I'm very confident that this type is meant to be used!

Reply:
I confirmed that it is used. We do say that a `GitSignature` can be returned, and this implements that interface.